pkgs/tools/misc/papis/default.nix

```diff
@@ -1,52 +1,48 @@
-{ lib, fetchFromGitHub, bashInteractive
-, python3, vim
+{ lib, fetchFromGitHub, fetchpatch
+, python36, xdg_utils
```

Comment:
I think it's cleaner to pass buildPythonApplication here and do papis = python36Packages.callPackage ../tools/misc/papis { }; in all-packages.nix

Reply:
That's discouraged because it doesn't allow the overriding of Python packages for specific applications.